ntoskrnl/include/internal/cc.h

```diff
@@ -309,9 +309,6 @@ CcMdlWriteComplete2(
     IN PMDL MdlChain
 );
 
-NTSTATUS
-CcRosFlushVacb(PROS_VACB Vacb, PIO_STATUS_BLOCK Iosb);
-
```

Comment:
The original kernel coding style appears to have been "use unique names rather than static functions". I don't think there's consensus either for or against that style, so please leave it alone.

Reply:
I took example after the 3 existing `static CcRos*()` in ntoskrnl/cc/view.c.
Obviously, I can remove that commit, meant to make code clearer, if you confirm it is unwanted.

Reply:
I prefer static functions. But that's no official coding style or whatever, so well...